resources/lang/en/accounts.php

```diff
@@ -18,6 +18,10 @@
             'new' => 'new email',
             'new_confirmation' => 'email confirmation',
             'title' => 'Email',
+            'locked' => [
+                '_' => 'Please contact :accounts if you need your email updated.',
+                'accounts' => 'osu! support',
```

Comment:
if it's `accounts@ppy.sh` as suggested in the linked issue then it should be this (see https://osu.ppy.sh/wiki/en/People/Account_support_team)

```suggestion
                '_' => 'Please contact the :accounts if you need your email updated.',
                'accounts' => 'account support team',
```